cookbooks/travis_ci_jvm/recipes/default.rb

```diff
@@ -22,12 +22,21 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-include_recipe 'travis_ci_standard'
+include_recipe 'travis_packer_templates'
+
+if node['travis_packer_templates']['env']['PACKER_BUILDER_TYPE'] == 'docker'
+  include_recipe 'travis_build_environment::hostname'
+else
+  include_recipe 'travis_ci_standard'
+end
+
 include_recipe 'java'
 include_recipe 'ant'
 include_recipe 'maven'
 include_recipe 'leiningen'
-include_recipe 'sbt-extras'
+# FIXME: sbt-extras installation currently busted due to incompatibility
+# something something https://github.com/travis-ci/travis-build/pull/701
+# include_recipe 'sbt-extras'
```

Comment:
As mentioned in https://github.com/travis-ci/travis-build/pull/701, as of paulp/sbt-extras#160 the problematic `addSbt` has been removed. So simply upgrading to a more current version of sbt-extras would remove that problem.